mobile/src/screens/marketplace.js

```diff
@@ -360,18 +366,26 @@ class MarketplaceScreen extends Component {
       if (url.hostname === 'twitter.com') {
         if (url.pathname === '/intent/tweet') {
           // Natively Tweet on Android
-          this.openNativeDeepLink(
+          const success = this.openNativeDeepLink(
```

Comment:
If we clean up the if/else logic in this method we can probably just return straight from the `this.openNativeDeepLink` call in each case.

Reply:
Let me try this.